lib/lib-markdown/src/main/java/org/sagebionetworks/markdown/MarkdownDaoImpl.java

```diff
@@ -1,27 +1,33 @@
 package org.sagebionetworks.markdown;
 
+import com.amazonaws.services.lambda.AWSLambda;
+import com.amazonaws.services.lambda.model.AWSLambdaException;
+import com.amazonaws.services.lambda.model.InvokeRequest;
+import com.amazonaws.services.lambda.model.InvokeResult;
 import org.json.JSONException;
 import org.json.JSONObject;
 import org.springframework.beans.factory.annotation.Autowired;
 
+import java.nio.charset.StandardCharsets;
+
 public class MarkdownDaoImpl implements MarkdownDao{
```

Comment:
This should be refactored to use the @Service annotation rather than an xml bean